airflow/providers/apprise/hooks/apprise.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, apprise_conn_id: str = default_conn_name) -> None:
 
     def get_config_from_conn(self):
         conn = self.get_connection(self.apprise_conn_id)
-        return json.loads(conn.extra_dejson["config"])
+        return conn.extra_dejson["config"]
```

Comment:
This would be a breaking changes, what if we keep previous behaviour, e.g. if `conn.extra_dejson["config"]` is string than try to `json.loads` and raise a warning?

Reply:
In think the change is too small to warn the user. It's better to keep both behaviour like this:

```python
config = conn.extra_dejson["config"]
return config if isinstance(config, dict) else json.loads(config)
```